lab.R

```diff
@@ -49,23 +53,30 @@ dataf = read_csv(file.path('data', 'ames.csv'))
 #' 
 #' 1. The paper abstract (see above) reports 2930 rows.  How many observations (rows) are in our version of the dataset?  
 #' 
-problem2.1 = 1.7e15 # scientific notation: 1.7 x 10^15
+problem2.1 = 2930 # scientific notation: 1.7 x 10^15
+
+skimr::skim(dataf)
+#the skimr dunct gives a print out that says 293- is there a functional p[rogramming way of accessing that part of the returned tibble to set it as the answer?
+
+
 
 #' 2. The abstract also reports 80 "explanatory variables (23 nominal, 23 ordinal, 14 discrete, and 20 continuous)."  In R, factors are used for both nominal and ordinal; and numerics are used for both discrete and continuous. How many of each are in our version?
 #' 
-problem2.2.factors = 7
-problem2.2.characters = 18000
-problem2.2.numerics = 12
+#' 
+#' for the number of factors it says there are 81 columns, I am assuming one of those is the index column and thus we have 80 variables. its hard to say though looking at skimr output. 
+problem2.2.factors = 80
```

Comment:
"Factor" is a specific type of variable in R, like "numeric" and "character" are types